AAI/AAIConnectProfile.md

```diff
@@ -339,18 +378,18 @@ the Broker.
             that may be reachable by a Claim Clearinghouse.
 
         4.  Visa Issuer MUST support public-facing
-            /userinfo endpoint. When presented with a valid Visa Access
-            Token, the /userinfo endpoint MUST return a success status
+            `/token` or `/userinfo` endpoint. When presented with a valid Visa Access
```

Comment:
I am not completely clear on the context of why we required visa issuers to implement a user info endpoint so I might be missing something, but I don't the use case we've been discussing for the 1.2 release requires this, so I'd suggest dropping it.

Reply:
@mbarkley This is the part where "Visa Access Tokens" are defined, so the visa is an access token and needs a userinfo endpoint to be exchanged for some useful data. But this whole section is confusing for me and probably not used by any implementation.

Reply:
Will add language to deprecate this section; will revert the language to previous state